[2024-10-17T09:46:18.555+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_from_local_to_postgres.extract_data manual__2024-10-17T09:46:16.373836+00:00 [queued]>
[2024-10-17T09:46:18.569+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_from_local_to_postgres.extract_data manual__2024-10-17T09:46:16.373836+00:00 [queued]>
[2024-10-17T09:46:18.570+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-10-17T09:46:18.592+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): extract_data> on 2024-10-17 09:46:16.373836+00:00
[2024-10-17T09:46:18.600+0000] {standard_task_runner.py:60} INFO - Started process 606 to run task
[2024-10-17T09:46:18.603+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl_from_local_to_postgres', 'extract_data', 'manual__2024-10-17T09:46:16.373836+00:00', '--job-id', '660', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpimv9oudg']
[2024-10-17T09:46:18.605+0000] {standard_task_runner.py:88} INFO - Job 660: Subtask extract_data
[2024-10-17T09:46:18.671+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_from_local_to_postgres.extract_data manual__2024-10-17T09:46:16.373836+00:00 [running]> on host 9c093e78d1a2
[2024-10-17T09:46:18.782+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_from_local_to_postgres' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2024-10-17T09:46:16.373836+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-17T09:46:16.373836+00:00'
[2024-10-17T09:46:18.848+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-10-17T09:46:18.864+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=etl_from_local_to_postgres, task_id=extract_data, execution_date=20241017T094616, start_date=20241017T094618, end_date=20241017T094618
[2024-10-17T09:46:18.939+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-10-17T09:46:18.981+0000] {taskinstance.py:3312} INFO - 1 downstream tasks scheduled from follow-on schedule check
